### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by SashaWeinstein**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/data-loading.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+name: Data Loading
+about: Track progress prepping source data
+title: "{Version} Facilites DB Dataloading"
+labels: ''
+assignees: ''
+
+---
+
+# FacDB Source Data Updates 
+Like most of our data products, source data must be updated in data library before facDB is run. As there are are many source datasets with varied update processes, this issue template should be opened to track progress towards updating all source data
```
Comment (AmandaDoyle) [inline .github/ISSUE_TEMPLATE/data-loading.md]: Update facDB to FacDB

File: .github/ISSUE_TEMPLATE/data-loading.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+- [ ] nypl_libraries
+Source: Scrape from NYPL website
+Source url: https://www.nypl.org/locations/list
+
+- [ ] uscourts_courts
+Source: Court locator for NY state
+Source url: http://www.uscourts.gov/court-locator/city/New%20York/state/NY
+
+## Source data from OpenData 
+To see if a dataset needs to be uploaded, check date last updated  in open data against version in recipes
```
Comment (AmandaDoyle) [inline .github/ISSUE_TEMPLATE/data-loading.md]: Should recipes be updated to libraries?

**Review by AmandaDoyle — COMMENTED** [review]
Have two minor comments.  Otherwise looks great.

**Review by mbh329 — APPROVED** [review]
This looks good, as we work through the data loading process we can add to this for any additional nuances (especially with any manual data loading)

**Pull request merged**